### PR TITLE
Add getSelectedValue to SelectListItem

### PR DESCRIPTION
**Pull request by D3fau4**
#56

### CODE REVIEW

File: library/include/borealis/list.hpp [hunk 1]
```diff
@@ -123,6 +123,7 @@ class SelectListItem : public ListItem
     SelectListItem(std::string label, std::vector<std::string> values, unsigned selectedValue = 0, std::string description = "");
 
     void setSelectedValue(unsigned value);
+    int getSelectedValue();
```
Comment (natinusala) [inline library/include/borealis/list.hpp:126]: Can you replace that by `unsigned` please?

**Review by natinusala — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: library/include/borealis/list.hpp (hunk 1 above)
Comment (D3fau4) [reply library/include/borealis/list.hpp]: done

**Comment by natinusala** [comment]
I see that selectedValue isn't initialized in the class, that's my fault, but can you add `= 0;` to the PR while you are here?

**Comment by D3fau4** [comment]
Okay, I think it's done.

**Comment by natinusala** [comment]
Great thank you!

**Review by natinusala — APPROVED** [review]

**Pull request merged**